functions/__fzf_display_value_or_error.fish

```diff
@@ -1,7 +1,7 @@
 # helper function for __fzf_search_shell_variables
 function __fzf_display_value_or_error --argument-names variable_name --description "Displays either the value of the variable passed in, or an informative message if it is not available."
     if set --query $variable_name
-        echo $$variable_name
+        set -S $variable_name | string replace -r "^\\\$$variable_name(: )?" ""
```

Comment:
Can you use `--show` instead of -S, please?

Reply:
What's the point of the call to string replace?

Reply:
> Can you use `--show` instead of -S, please?

Sure thing!

> What's the point of the call to string replace?

Make the output more concise. This is the unmodified output of `set --show`:

![](https://user-images.githubusercontent.com/44045911/103395784-e37a6d00-4b6a-11eb-8399-6312d8b455a8.png)

Reply:
Nice, that's very clever, much easier to read indeed